cdap-app-fabric/src/main/java/io/cdap/cdap/app/store/Store.java

```diff
@@ -276,6 +276,15 @@ Map<ProgramRunId, RunRecordDetail> getRuns(ApplicationId applicationId, ProgramR
    */
   Map<ProgramRunId, RunRecordDetail> getActiveRuns(ApplicationId applicationId);
 
+  /**
+   * Fetches the active (i.e STARTING or RUNNING or SUSPENDED) run records against a given ApplicationId.
```

Comment:
given ApplicationId -> for the latest version of the given application

Reply:
Should also document what it does if the application doesn't exist (I would assume it returns an empty map?)

Reply:
done